### PR TITLE
Cleanup dockerfiles.

### PR DESCRIPTION
**Pull request by rabbah**

### CODE REVIEW

File: core/nodejsActionBase/Dockerfile [hunk 1]
```diff
@@ -15,17 +15,30 @@
 # limitations under the License.
 #
 
-FROM buildpack-deps:trusty-curl
-
-ENV DEBIAN_FRONTEND noninteractive
+FROM node:stretch
```
Comment (dgrove-oss) [inline core/nodejsActionBase/Dockerfile:18]: Are we ok with this floating to whatever the latest node version is, or should we bind it a little tighter?

Comment (rabbah) [reply core/nodejsActionBase/Dockerfile:18]: we don't actually build or publish this image it's primarily used as a template. So I was thinking this is ok in that context.

**Comment by rabbah** [comment]
@dgrove-oss do you want to look again I piled on a few more commits.

File: .travis.yml [hunk 2]
```diff
@@ -43,6 +43,7 @@ deploy:
       repo: apache/openwhisk-runtime-nodejs
   - provider: script
     script: "./tools/travis/publish.sh openwhisk 8 nightly && ./tools/travis/publish.sh openwhisk 10 nightly && ./tools/travis/publish.sh openwhisk 12 nightly && ./tools/travis/publish.sh openwhisk 14 nightly"
+    script: "./tools/travis/publish.sh openwhisk nodejs10Action nightly && ./tools/travis/publish.sh openwhisk nodejs12Action nightly && ./tools/travis/publish.sh openwhisk nodejs14Action nightly && ./tools/travis/publish.sh openwhisk typescript37Action nightly"
```
Comment (dgrove-oss) [inline .travis.yml:46]: I think you wanted to delete line 45 and replace with line 46.  Not have both line 45 and line 46.

**Comment by rabbah** [comment]
@dgrove-oss fixed the bug - thanks for catching it. Appears I cannot merge without you reapproving the pr.

**Pull request merged**